### ISSUE TITLE
Feature Request: Users overview

### ISSUE THREAD

**Issue by Txikivasco** [saltcorn/saltcorn#1965]
I've been thinking that the users panel is kind of lacking default features. For instance, I would like to see when registered users log in the app.

The solution could be something like a join table for each user that automatically registers a "date" field on a log in/log off event and maybe display on the users pannel the last time a user logged in.

**Comment by Txikivasco** [comment]
But I have to add an event for each user manually?

**Comment by cfredericksen** [comment]
@Txikivasco 
No, you can use the user.id for the context. Here is my trigger.
```
	var date = new Date();
	var current_date = date.getFullYear()+"-"+(date.getMonth()+1)+"-"+ date.getDate();
	var current_time = date.getHours()+":"+date.getMinutes()+":"+ date.getSeconds();
	var date_time = current_date+", "+current_time;	
console.log(date_time); // 👉️ "7-25-2023"
for (var i = 0; i < 1; i++) {
    await Table.findOne({name: 'users'}).tryUpdateRow(({lastlogin: date_time}), user.id);
}
```

**Comment by pyhedgehog** [comment]
As soon as `users` is (almost) regular table, you can create view with everything you want. 😄

**Comment by pyhedgehog** [comment]
@cfredericksen
Please don't do like this. Just define `lastlogin` field type as Date and insert `new Date()` there.

**Comment by cfredericksen** [comment]
> @cfredericksen
> Please don't do like this. Just define `lastlogin` field type as Date and insert `new Date()` there.
> 
> @glutamate
> Catching (and logging) of `Login` event will work on your own saltcorn instance, but not on saltcorn.com tenant.

@pyhedgehog 
I couldn't get it working on my self-hosted Saltcorn tenant. Is there a reason I shouldn't do it this way? I have an update and an insert event that occurs during login.

**Comment by Txikivasco** [comment]
> @glutamate Catching (and logging) of `Login` event will work on your own saltcorn instance, but not on saltcorn.com tenant.

And that's precisely the reason why it needs to be a default feature, so we don't need any workaround.

**Comment by cfredericksen** [comment]
@pyhedgehog 
I updated the login trigger. I ran into search errors with the previous code.

```
var now = Date()
var current = new Date(now).toISOString();
for (var i = 0; i < 1; i++) {
    await Table.findOne({name: 'users'}).tryUpdateRow(({lastlogin: current}), user.id);
}
```
